app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+import os
+
+
+def move_clear_file(file_path1: str, file_path2: str) -> None:
+    with (
+        open(file_path1, "r") as file_in,
+        open(file_path2, "w") as file_out
+    ):
+        file_out.write(file_in.read())
+        file_in.close()
+        os.remove(file_path1)
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("This is not valid")
+    command, file_path_name1, file_path_name2 = command.split()
+    if command != "mv":
+        raise ValueError(
+            f'"{command}" This is an invalid command!'
+            f' The correct one is: "mv"'
+        )
+
+    path = os.path.join(file_path_name2)
```

Comment:
Why you need to call this if there is only 1 path to be joined?

Reply:
In order for the path to be correct for all OS.